_timeline/2015-01-01-Hyberboria.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Hyperboria
+date: 2015-01-01
+---
+Toronto-based individuals are involved with the international cjdns mesh networking community--[Hyperboria](https://hyperboria.net/).
```

Comment:
You're my first Hype peer so whenever that happened? We can revise this date to June 2013 :)